include/octo-logger-cpp/log.hpp

```diff
@@ -71,14 +78,15 @@ class Log
     }
 
     template <typename... T>
-    void formatted(fmt::format_string<T...> fmt, T&&... args)
+    Log& formatted(fmt::format_string<T...> fmt, T&&... args)
     {
         if (stream_)
         {
             *stream_ << fmt::format(fmt, std::forward<T>(args)...);
         }
+        return *this;
```

Comment:
why is this change needed? the formatted is for a single ofrmatted log, that can contain all the log string in it

Reply:
reverted